src/accessors/external_accessor.ts

```diff
@@ -262,6 +271,65 @@ export class ExternalAccessor implements IManagementApiAccessor {
     await this._httpClient.post<UserTaskResult, any>(url, userTaskResult, requestAuthHeaders);
   }
 
+  // ManualTasks
+  public async getManualTasksForProcessModel(identity: IIdentity, processModelId: string): Promise<ManualTaskList> {
+
+    const requestAuthHeaders: IRequestOptions = this._createRequestAuthHeaders(identity);
+
+    let url: string = restSettings.paths.processModelManualTasks.replace(restSettings.params.processModelId, processModelId);
+    url = this._applyBaseUrl(url);
```

Comment:
Nie variablen mutieren!

```suggestion
    const url = this._applyBaseUrl(urlRestPart);
```